app/lib/tool/neat_task/pub_dev_tasks.dart

```diff
@@ -27,6 +28,14 @@ void setupFrontendPeriodicTasks() {
     task: backfillNewFields,
   );
 
+  // Checks the Datastore integrity of the model objects.
+  _weekly(
```

Comment:
Do we ensure that all weekly tasks don't run at the same time?

Reply:
Yeah, `package:neat_periodic_task` does the heavy lifting there, and we have a Datastore-based state store for it.